src/style/button/props-button.tsx

```diff
@@ -0,0 +1,43 @@
+import "./button.css"
+import { colors } from "../colors";
+import { components } from "../typography/typography";
+import { styleButton } from "./style-button";
+
+
+interface ButtonProps {
+  variant: 'primary' | 'secondary' | 'cta';
+  children: React.ReactNode;
+  onClick?: () => void;
+  disabled?: boolean;
+  icon?: React.ReactNode;
+  height?: "default" | "defaultCompact";
```

Comment:
Aqui uma estratégia que pode simplificar a clareza do código é vc usar um boolean opcional chamado `compact`, por exemplo. Se essa prop não for passada ou for `false`, o botão fica com a altura padrão.

Reply:
entendi, vou corrigir isso